libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -109,6 +109,51 @@ void AC_AutoTune::stop()
     // we expect the caller will change the flight mode back to the flight mode indicated by the flight mode switch
 }
 
+// Autotune aux function trigger
+void AC_AutoTune::do_aux_function(const RC_Channel::AuxSwitchPos ch_flag)
+{
+    if (mode != TuneMode::SUCCESS) {
+        if (ch_flag == RC_Channel::AuxSwitchPos::HIGH) {
+            gcs().send_text(MAV_SEVERITY_NOTICE,"AutoTune: must be complete to test gains");
+        }
+        return;
+    }
+
+    switch(ch_flag) {
+        case RC_Channel::AuxSwitchPos::LOW:
+            // load original gains
+            load_gains(GainType::GAIN_ORIGINAL);
+            update_gcs(AUTOTUNE_MESSAGE_TESTING_END);
+            break;
+        case RC_Channel::AuxSwitchPos::MIDDLE:
+            // Middle position is unused for now
```

Comment:
This is where we could change to mode AutoTune if the tune has not been completed.

********* For Discussion *********